Assets/Mirror/Examples/Lobby/Scripts/OfflineGUI.cs

```diff
@@ -26,6 +26,7 @@ void OnGUI()
 
             if (GUILayout.Button("Join Game"))
             {
+                QualitySettings.vSyncCount = 2;
```

Comment:
Same,  this does not belong here

Reply:
This is a git goof on my part - the change is already live.  It got picked up by accident when I made the branch.